types/lru-cache/index.d.ts

```diff
@@ -1,19 +1,125 @@
-// Type definitions for lru-cache 4.1
+// Type definitions for lru-cache 5.1
```

Comment:
Please follow instructions [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version) for major version updates.